bam/inc/com/centreon/broker/bam/impact_values.hh

```diff
@@ -33,31 +33,37 @@ namespace bam {
  *  downtime impact, acknowledgement impact.
  */
 class impact_values {
+ public:
+  typedef enum {
+    state_ok,
```

Comment:
When calling this enum value, we have to write this right ?
`state.state_ok`

I would suggest this syntaxe instead :
`state.ok`

Reply:
state.ok would not be accepted by the compiler. And you can use directly state_ok in almost all cases.

Reply:
Ok, it may be naive.

But with ctrl+F in the PR we can see the following :
"impact_values::state_" is used 1 time
"ba::state::state_" is used 53 times

Which makes sence, since a backend interface will always be more used by clients than by the backend itself. 
Principle being :
>  Always design your code for client perspective

Reply:
all other state_ are typedefed from impact_values, so we can get the same type everywhere.